validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+# RANCHERINT Schemas
+
+## Test Suite: Longhorn
+
+### Fresh Longhorn Installation via Rancher App Catalog
+
+Verify Longhorn can be successfully installed through Rancher's app catalog with default settings on a clean Kubernetes cluster.
+
+| Step Number | Action                                            | Data | Expected Result                                             |
+|------|---------------------------------------------------|------|-------------------------------------------------------------|
+| 1 | Log into Rancher UI with cluster admin privileges |      | Rancher dashboard loads successfully                        |
```

Comment:
I would put this down as a prerequisite to the test, i.e. https://github.com/rancher/tests/blob/main/validation/harvester/schemas/harvester_schemas.md#snapshot--restore-of-a-downstream-cluster

Reply:
Does the Qase importer treat Actions with Prequisite differently?